README.md

```diff
@@ -1,24 +1,97 @@
+You can also help make the docs awesome.
+Your feedback is welcome.
+Your writing, editing, translating, designing, and developing skills are welcome.
+You being a part of our community is welcome.
 
-## Deploying
+## Chat with Us
 
-The site is automatically deployed when commits land in `latest`, via Netlify.
+You can learn more about Astro, get support, and meet other devs in [our Discord community](https://astro.build/chat).
 
-The "next" docs are automatically deployed when commits land in `main`, via Netlify at <https://main--astro-docs-2.netlify.app/getting-started/>.
+## Raise an Issue
+
+Is something missing?
+Is something wrong?
+Could something be better?
+Issues are a quick way for you to offer us feedback about the docs.
+
+Before you share, please [see if your issue has already been reported](https://github.com/withastro/docs/issues).
+
+## Edit a Page
+
+Every page on [docs.astro.build](https://docs.astro.build/) has an **Edit this page** button in the sidebar.
+You can click that button to edit the source code for that page in **GitHub**.
+
+After you make your changes, click **Commit changes**.
+This will automatically create a [fork](https://docs.github.com/en/pull-requests/collaborating-with-pull-requests/working-with-forks/about-forks) of the docs in your GitHub account with the changes.
+
+Once your edits are ready in GitHub, follow the prompts to **create a pull request** and submit your changes for review.
+Every pull request needs to be reviewed by our contributors.
+
+## Develop
+
+To begin developing locally, checkout this project from your machine.
+
+```shell
+git clone git@github.com:withastro/docs.git
+```
+
+You can install and run the project locally using your preferred package manager.
+
+```shell
+npm install
+
+npm start
+```
+
+If you’re copying these instructions, remember to [configure this project as a fork](https://docs.github.com/en/pull-requests/collaborating-with-pull-requests/working-with-forks/configuring-a-remote-for-a-fork).
+
+```shell
+git remote add upstream git@github.com:csstools/docs.git
+```
+
+At any point, create a branch for your contribution.
+We are not strict about branch names.
+
+```shell
+git checkout -b add/klingon-language
+```
+
+That’s it.
+As you [open a pull request](https://github.com/withastro/astro/compare), please include a clear title and description.
+
+```markdown
+# Add Klingon language to Getting Started page
+
+This adds the Klingon language and also updates the sidebar and language selection components.
+```
+
+Thank you for helping make the docs awesome.
+And please, [come chat with us](https://astro.build/chat) if you have any questions.
+
+## Deploy
+
+Every pull request generates a preview using **Netlify** for anyone to see.
+
+Use the **Deploy Preview** of your pull request to review and share your changes.
+
+The docs site will be automatically updated whenever pull requests are merged.
+
+## Next Steps
+
```

Comment:
Do we want to include an extra step here, re: joining Discord, and/or *specifically* the #docs channel?

Reply:
I’m unsure what that would look like. I would want to defer that to another improvement, or... a separate PR from our contributors. 😉

Reply:
Oh, I just meant adding it to the list of steps as one additional item. But yes, that can come separately.  :)